lib/qtermwidget.cpp

```diff
@@ -698,6 +698,20 @@ void QTermWidget::setBlinkingCursor(bool blink)
     m_impl->m_terminalDisplay->setBlinkingCursor(blink);
 }
 
+void QTermWidget::setBidiEnabled(bool enabled)
+{
+    if (!m_impl->m_terminalDisplay)
+        return;
+    m_impl->m_terminalDisplay->setBidiEnabled(enabled);
+}
+
+bool QTermWidget::isBidiEnabled()
+{
+    if (!m_impl->m_terminalDisplay)
+        return false; // Default value
```

Comment:
Now we have a better default :)

Reply:
No, this was taken from the widget itself and not the terminal, here https://github.com/lxde/qtermwidget/blob/master/lib/TerminalDisplay.cpp#L333

I don't know, is it fine for the widget to have that “on” by default as well?

Reply:
Oh, you want to change the default in the termwidget as well? I don't think we need that really. Apps using it can enable or disable as they want.